src/higley_lab_to_nwb/benisty_2022/benisty_2022nwbconverter.py

```diff
@@ -4,7 +4,7 @@
 from higley_lab_to_nwb.benisty_2022.benisty_2022_spike2signals_interface import Benisty2022Spike2SignalsInterface
 from higley_lab_to_nwb.benisty_2022.benisty_2022_imaginginterface import Benisty2022ImagingInterface
 
-from neuroconv.datainterfaces import VideoInterface
+from neuroconv.datainterfaces import VideoInterface, FacemapInterface
```

Comment:
can you also pin the version of neuroconv you're using here in the requirements.txt file?  you can pin it temporarily to the branch you're working on.

Reply:
Yes, thank you for noticing it! I also pinned the requirement for the facemap extension